code/game/machinery/computer/communications.dm

```diff
@@ -41,6 +41,9 @@
 	/// The last lines used for changing the status display
 	var/static/last_status_display
 
+	/// The next time the security level can be changed from this console
+	var/next_lvl_change = 0
```

Comment:
Use proper cooldown defines

Reply:
0 SECONDS?

Reply:
no, search for COOLDOWN_DECLARE and such

Reply:
oh wow thats neat

Reply:
im an idiot theres already a thing here